rtf_vis_tool/src/Components/LandingPageGraph.js

```diff
@@ -130,23 +125,39 @@ function LandingPageGraph() {
             </div>
 
             {/* Render popups only when the respective node is clicked. */} 
-            {showDA_PC && <Bundle_Adj_PC toggleBA_PC={toggleBundleAdj_PointCloud}/>}
+            {showDA_PC && <PCViewer title={'Data Association Point Cloud'} 
+                                    togglePC={toggleDA_PointCloud} 
+                                    filePath={'results/ba_input/points3D.txt'}/>}
+            {showBA_PC && <PCViewer title={'Bundle Adjustment Point Cloud'}
+                                    togglePC={toggleBA_PointCloud}
+                                    filePath={'results/ba_output/points3D.txt'}/>}
             {showFS && <FrontendSummary json={fs_json} toggleFS={toggleFrontEndSummaryDisplay}/>}
             {showMVO && <MVOSummary json={mvo_json} toggleMVO={toggleMVOMetrics}/>}
 
             <div className="gtsfm_graph">
 
-                {/* Render all Gray and Blue Nodes (43 combined). */}
+                {/* Render basic Gray and Blue Nodes (41 combined). */}
```

Comment:
just curious -- why 41 instead of 43 now?

Reply:
So I am still rendering 43 nodes total. But the 2 that are not rendered in the for loop are the nodes `Data Association GtsfmData` and `Bundle Adjustment GtsfmData`. I wasn't able to get these two render within the for loop since they have the custom functionality of displaying the 2 point clouds. So 41 are rendered in the generic loop + 2 nodes that display custom point clouds.

Reply:
cool -- maybe you can just add that as an inline comment?

Reply:
yup.